magento/magento2
# Remove references to files from extjs which got removed in e1ccdb…

…83e7e312572038c476d0695300700db849

### Description (*)
While running Magento 2.4-develop locally, I noticed some errors both in my brower's console and the `var/log/system.log` file:

```
[2024-07-18T14:31:10.013255+00:00] main.ERROR: Compilation from source: LESS file is empty: adminhtml/Magento/backend/en_US/extjs/resources/css/ytheme-magento.less [] []
[2024-07-18T14:31:10.013254+00:00] main.ERROR: Compilation from source: LESS file is empty: adminhtml/Magento/backend/en_US/extjs/resources/css/ext-all.less [] []
```

After some searching around, it looks like the entire directory `lib/web/extjs` got removed in e1ccdb83e7e312572038c476d0695300700db849

It looks like @Atul-glo35265 didn't notice there were still references to some of those files in the codebase, which were still being used in the backoffice.

This PR cleans up those references.

The removal of the phtml file is because it also referenced extjs, so it would also no longer work, and it seems the file is not being used (at least not in Magento OS).
From [the commit message](https://github.com/magento/magento2/commit/698ab5c4ccc) that introduced the file, I think this may have been used in Adobe Commerce since they mention a Banner Rotation Widget, which doesn't exist in Magento OS. But since this commit is 9 years old, I have no idea if that still exists. And maybe it was also refactored recently in Adobe Commerce to no longer use the extjs files?
Maybe somebody with access to Adobe Commerce source code can double check this?

### Related Pull Requests

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
1. Open the backoffice
2. See if there are errors in your browsers console about trying to load css files from extjs
3. See in your `var/log/system.log` if there are errors about empty less files from extjs

### Questions or comments
Preferably, this PR should be included in whatever next Magento release (2.4.8?) together with the changes from e1ccdb83e7e312572038c476d0695300700db849

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#38960: Remove references to files from extjs which got removed in e1ccdb…
